src/agentscope/models/dashscope_model.py

```diff
@@ -294,9 +313,13 @@ def generator() -> Generator[str, None, None]:
                 kwargs,
                 response,
             )
+            if self.api_type == "application":
```

Comment:
Similar as above, consider to use enum for `api_type`

Reply:
Now, only `application` is used, maybe w'll use enum later when more APIs are supported.